go-app-ussd_popi_rapidpro.js

```diff
@@ -301,6 +301,18 @@ go.app = function() {
                 });
         });
 
+        self.states.add("state_msisdn_change_completed", function(name) {
```

Comment:
Adding this state to give the flow a chance to do the actual switch, then back to state start to get the updated contact.